qa/L0_inferentia_perf_analyzer/test.sh

```diff
@@ -77,21 +83,83 @@ if [ ${USE_TENSORFLOW} == "1" ]; then
         --compiled_model $PWD/add_sub_model_tf1 \
         --triton_input INPUT__0,INT64,-1x4 INPUT__1,INT64,-1x4 \
         --triton_output OUTPUT__0,INT64,-1x4 OUTPUT__1,INT64,-1x4 \
-        --triton_model_dir models_single_tf1/add-sub-1x4 \
+        --triton_model_dir models_single_tf1_default_batch/add-sub-1x4 \
         --neuron_core_range 0:0
     python ${TRITON_PATH}/python_backend/inferentia/scripts/gen_triton_model.py \
         --model_type "tensorflow" \
         --compiled_model $PWD/add_sub_model_tf1 \
         --triton_input INPUT__0,INT64,-1x4 INPUT__1,INT64,-1x4 \
         --triton_output OUTPUT__0,INT64,-1x4 OUTPUT__1,INT64,-1x4 \
-        --triton_model_dir models_multiple_tf1/add-sub-1x4 \
+        --triton_model_dir models_multiple_tf1_default_batch/add-sub-1x4 \
         --triton_model_instance_count 3 \
         --neuron_core_range 0:7
+    python ${TRITON_PATH}/python_backend/inferentia/scripts/gen_triton_model.py \
+        --model_type "tensorflow" \
+        --max_batch_size 1000 \
+        --preferred_batch_size 8 \
+        --compiled_model $PWD/add_sub_model_tf1 \
+        --triton_input INPUT__0,INT64,-1x4 INPUT__1,INT64,-1x4 \
+        --triton_output OUTPUT__0,INT64,-1x4 OUTPUT__1,INT64,-1x4 \
+        --triton_model_dir models_single_batched_tf1_default_batch/add-sub-1x4 \
+        --neuron_core_range 0:0
+    python ${TRITON_PATH}/python_backend/inferentia/scripts/gen_triton_model.py \
+        --model_type "tensorflow" \
+        --max_batch_size 1000 \
+        --preferred_batch_size 8 \
+        --compiled_model $PWD/add_sub_model_tf1 \
+        --triton_input INPUT__0,INT64,-1x4 INPUT__1,INT64,-1x4 \
+        --triton_output OUTPUT__0,INT64,-1x4 OUTPUT__1,INT64,-1x4 \
+        --triton_model_dir models_multiple_batched_tf1_default_batch/add-sub-1x4 \
+        --triton_model_instance_count 3 \
+        --neuron_core_range 0:7
+
+    python ${TRITON_PATH}/python_backend/inferentia/scripts/gen_triton_model.py \
+        --model_type "tensorflow" \
+        --compiled_model $PWD/add_sub_model_tf1 \
+        --triton_input INPUT__0,INT64,-1x4 INPUT__1,INT64,-1x4 \
+        --triton_output OUTPUT__0,INT64,-1x4 OUTPUT__1,INT64,-1x4 \
+        --triton_model_dir models_single_tf1_no_batch/add-sub-1x4 \
+        --neuron_core_range 0:0\
+        --disable_batch_requests_to_neuron
+    python ${TRITON_PATH}/python_backend/inferentia/scripts/gen_triton_model.py \
```

Comment:
A lot of these args are repeated. Why not re-use a template and add extra args instead of always using the entire (long) command.